packages/iTwinUI-react/src/core/utils/components/Resizer.tsx

```diff
@@ -0,0 +1,298 @@
+          }
+          const newWidth = initialWidth + (initialPointerX - clientX);
+          if (newWidth >= minWidth) {
+            width = elementRef.current.style.width = `${newWidth}px`;
+            translateX = initialTranslateX - (initialPointerX - clientX);
+          }
+          elementRef.current.style.transform = `translate(${translateX}px, ${translateY}px)`;
+          break;
+        }
+        case 'top': {
+          const newHeight = initialHeight + (initialPointerY - clientY);
+          if (newHeight < minHeight) {
+            break;
+          }
+          height = elementRef.current.style.height = `${newHeight}px`;
+          translateY = initialTranslateY - (initialPointerY - clientY);
+          elementRef.current.style.transform = `translate(${translateX}px, ${translateY}px)`;
+          break;
+        }
+        case 'top-right': {
+          const newHeight = initialHeight + (initialPointerY - clientY);
+          if (newHeight >= minHeight) {
+            height = elementRef.current.style.height = `${newHeight}px`;
+            translateY = initialTranslateY - (initialPointerY - clientY);
+          }
+          width = elementRef.current.style.width = `${
+            initialWidth - (initialPointerX - clientX)
+          }px`;
+          elementRef.current.style.transform = `translate(${translateX}px, ${translateY}px)`;
+          break;
+        }
+        case 'right': {
+          width = elementRef.current.style.width = `${
+            initialWidth - (initialPointerX - clientX)
+          }px`;
+          break;
+        }
+        case 'bottom-right': {
+          width = elementRef.current.style.width = `${
+            initialWidth - (initialPointerX - clientX)
+          }px`;
+          height = elementRef.current.style.height = `${
+            initialHeight - (initialPointerY - clientY)
+          }px`;
+          break;
+        }
+        case 'bottom': {
+          height = elementRef.current.style.height = `${
+            initialHeight - (initialPointerY - clientY)
+          }px`;
+          break;
+        }
+        case 'bottom-left': {
+          const newWidth = initialWidth + (initialPointerX - clientX);
+          if (newWidth >= minWidth) {
+            width = elementRef.current.style.width = `${newWidth}px`;
+            translateX = initialTranslateX - (initialPointerX - clientX);
+          }
+          height = elementRef.current.style.height = `${
+            initialHeight - (initialPointerY - clientY)
+          }px`;
+          elementRef.current.style.transform = `translate(${translateX}px, ${translateY}px)`;
+          break;
+        }
+        case 'left': {
+          const newWidth = initialWidth + (initialPointerX - clientX);
+          if (newWidth < minWidth) {
+            break;
+          }
+          width = elementRef.current.style.width = `${newWidth}px`;
+          translateX = initialTranslateX - (initialPointerX - clientX);
```

Comment:
maybe save the `initialPointerX - clientX` part in a variable for readability and reuse

Reply:
Moved to a variable